### ISSUE TITLE
Cannot specify http IONOS_API_URL

### ISSUE THREAD

**Issue by maboehm** [ionos-cloud/terraform-provider-ionoscloud#154]
## Description
https://github.com/ionos-cloud/terraform-provider-ionoscloud/pull/137 (Version `5.2.19` changed the URL parsing behavior, such that http URL no longer work (https://github.com/ionos-cloud/terraform-provider-ionoscloud/blame/master/vendor/github.com/ionos-cloud/sdk-go/v5/configuration.go#L244).

We currently need to specify a http connection behavior, because we need to go through a local proxy to reach the API.

```
provider "ionoscloud" {
  endpoint = "http://local-proxy/cloudapi/v5"
}
```

Results in:
```
an error occured while fetching a User ID <UUID> Get "https://http//local-proxy/cloudapi/v5/um/users/<UUID>?depth=10"
```

## Expected behavior
I expected the provider to respect my scheme (http), if I explicitly set it.

## Environment

Terraform version: 
```
 v0.15.4
```
Provider version: 
```
5.2.19-latest
```
OS:
```
linux
```

**Comment by cristiGuranIonos** [comment]
Thanks for letting us know, we will take a look.

**Comment by cristiGuranIonos** [comment]
Issue is fixed, we will provide a new release as soon as possible

**Comment by cristiGuranIonos** [comment]
Fixed in 5.2.22 and 6.0.0
